src/components/DpTootip/utils/tooltip.js

```diff
@@ -0,0 +1,100 @@
+import { arrow, computePosition, flip, offset, shift } from '@floating-ui/dom'
```

Comment:
This implementation adds all needed markup right on mounted, and leaves the nodes in the dom until destroyed. This may lead to perf issues with lots of tooltip instances within one view, eg. in large lists with tooltips in it.

In the `VTooltip` implementation, the dom nodes representing the toolips were added whenever a tooltip was opened, and then, they were removed from the dom ca. 30 seconds after being hidden. I guess this was to ensure that when somebody hovers/unhovers a tooltip multiple times, the dom manipulations would not stack up.

Reply:
good point.... I will rewrite that

Reply:
I tried to find a better solution here 94d01bcb47d5a69010ef6b0ff0fe85eefa86e94e

Reply:
This looks good, tooltip elements are now created onHover instead of onMounted. However the `handleTimeoutForDestroy` thing does not what it is supposed to be, the element is not removed from dom.

Reply:
no? In my tests it did remove them. Do You get any errors? (Its not removed instantly but after about 3  seconds after move out.)

Reply:
i'm testing it from within storybook.